freedomofpress/securedrop-client
# securedrop-client 0.0.13

# Description

* remove user refresh and replace with sync icon (#732)
* build-requirements: update for production beta (#730)
* No sync on ui operations (#721)
* Use SecureQLabel for message previews (#720)
* Show DD MMM format for source title (#719)
* Add new metadata queue. (#715)
* Improve performance of storage.get_remote_data (#709)
* app/queue: prioritize user-triggered state changes (#708)
* Fix HTML entities being escaped in speech bubbles. (#703)
* Activity indicator for file download / decryption. (#702)
* Rename VMs (#701)

# Test Plan

* Run the client. Make sure it still works.
* Build the Debian package in `securedrop-debian-packaging`.

# Checklist

If these changes modify code paths involving cryptography, the opening of files in VMs or network (via the RPC service) traffic, Qubes testing in the staging environment is required. For fine tuning of the graphical user interface, testing in any environment in Qubes is required. Please check as applicable:


If these changes add or remove files other than client code, packaging logic (e.g., the AppArmor profile) may need to be updated. Please check as applicable:
